docker/prod/Dockerfile-web

```diff
@@ -0,0 +1,18 @@
+FROM python:3.6-8-alpine3.9
```

Comment:
Should it be specific to this version, or just 3.6?

Reply:
There is an open discussion around this. 
The development image is not immutable so we might be happy to use the "latest" 3.6
For the prod image, builds must be absolutely repeatable and this implies locking all dependencies to a very specific version